web-animations/testcommon.js

```diff
@@ -56,3 +61,12 @@ function removeElement(element) {
 function type(object) {
     return Object.prototype.toString.call(object).slice(8, -1);
 }
+
+// Returns expected top of the target element at currentTime
+function getExpectedTop(currentTime, endTime) {
+    if (!endTime) {
+        endTime = ANIMATION_DURATION;
+    }
+    return ANIMATION_TOP_0 + (ANIMATION_TOP_1 - ANIMATION_TOP_0)
+        * (currentTime % endTime);
```

Comment:
It's not very clear you intend this to test expected iteration values, adding a line like `var iterationFraction = (currentTime / iterationDuration) % 1;` would make this more clear.
Should endTime be renamed iterationDuration? In the spec "end time" refers to the time at which a node finishes its active phase with respect to its parent's timeline which is quite different to how it's being used here.
Also KEYFRAMES doesn't define a linear animation, you'll need to incorporate ANIMATION_TOP_0_5 here as well.